README.md

```diff
@@ -0,0 +1,46 @@
+# External Secrets
+
+This operator reads information from a third party service
+like [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/)  and automatically injects the values as [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) (other secret targets are also planned).
```

Comment:
We can leave the `(other secret targets are also planned).` out, since we have a `Planned Features` section below.